kolumnextractor.py

```diff
@@ -28,10 +31,23 @@ def determine_read(info_dict):
             logging.error('File is not a CSV. Please supply a CSV file type.')
 
 
-def reading(filepath: str, columns):
-    return determine_read({"path": filepath,
-                           "columns": columns,
-                           "type": os.path.splitext(filepath)[1]})
+def reading(filepath: str, data_definition):
+    match type(data_definition).ToString():
+        case "Data":
```

Comment:
For now, given the Data and DataContainer classes aren't fully implemented we should only allow dict or lists.